js/main.js

```diff
@@ -32,6 +33,7 @@ const autoPromoteElement = document.getElementById('autoPromote');
 const autoLearnElement = document.getElementById('autoLearn');
 
 const jobTabButton = document.getElementById('jobTabButton');
+const gameOverMessage = document.getElementById('gameOverMessage');

```

Comment:
Bitte DOM Elemente immer auch so benennen, sonst kommt man durch einander was halt ein String und was ein HTMLElement ist.
```suggestion
const gameOverMessageElement = document.getElementById('gameOverMessage');

```